programs/ssl/ssl_client2.c

```diff
@@ -1663,13 +1696,28 @@ int main( int argc, char *argv[] )
     fflush( stdout );
 
     mbedtls_entropy_init( &entropy );
-    if( ( ret = mbedtls_ctr_drbg_seed( &ctr_drbg, mbedtls_entropy_func,
-                                       &entropy, (const unsigned char *) pers,
-                                       strlen( pers ) ) ) != 0 )
+    if (opt.reproducible)
     {
-        mbedtls_printf( " failed\n  ! mbedtls_ctr_drbg_seed returned -0x%x\n",
-                        -ret );
-        goto exit;
+        srand( 1 );
```

Comment:
`srand` is no longer needed.

Reply:
I guess `srand` is needed since `dummy_entropy` uses `rand`, is it not ?

Reply:
Oh right, my bad.